### PR TITLE
remove bayesian filtering

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn We need to run [classify_comments](https://github.com/praekelt/django-moderator/blob/feature/remove-bayesian-filtering/moderator/utils.py#L53-L57) in order to auto-remove user reported comments. I'm tempted to stick it in a cron job. Is there a way we can attach it to a signal? So that everytime a user reports a comment we classify it. It's a very inexpensive call since it just checks if a comment has received more than 3 `-1` votes. I don't think it warrants a celery task.

**Comment by smn** [comment]
Why not a celery task? I prefer celery tasks over cron any day, even for simple stuff.

File: moderator/models.py [hunk 1]
```diff
@@ -202,5 +188,13 @@ def realtime_comment_classifier(sender, instance, created, **kwargs):
             from moderator.utils import classify_comment
             classify_comment(instance)
 
+
+@receiver(object_liked)
+def flag_reported_comments(instance, request, **kwargs):
```
Comment (miltontony) [inline moderator/models.py:193]: @smn I decided to use the likes `signal` and `celery`.

**Comment by smn** [comment]
:+1: for ripping this out, forget my earlier bikeshed. Celery or cron either is good as long as it happens.

File: moderator/models.py (hunk 1 above)
Comment (smn) [inline moderator/models.py:193]: nice!

**Comment by miltontony** [comment]
Celery is a better fit. I just didn't want to have to use the celery scheduler or cron for that matter. After some digging I found that I could use the likes signal, which I could then combine with celery.

**Comment by miltontony** [comment]
@smn If you're happy with this branch, I plan on bumping the major version number to `1.0.0`

**Comment by smn** [comment]
yes I like how you put a :bomb: under the problem to solve it.
Please remember to tag the previous version so whoever wants that broken stuff can still refer to it.

**Pull request merged**